internal/operands/template-validator/resources.go

```diff
@@ -113,6 +117,7 @@ func newDeployment(namespace string, replicas int32, image string) *apps.Deploym
 	const volumeName = "tls"
 	const certMountPath = "/etc/webhook/certs"
 	trueVal := true
+	percentVal := intstr.FromString("25%")
```

Comment:
Is this value taken from somewhere? Or how do you know this is correct value?

Reply:
All defaults are from the documentation that is on the struct fields.
I have ale checked it in OCP 4.5.7 cluster.

Reply:
Unfortunately, the code that sets these defaults is part of the `k8s.io/kubernetes` repo, which cannot be easily imported as module.